scopes/component/graph/ui/dependencies-graph/dep-edge/dep-edge.tsx

```diff
@@ -12,3 +13,16 @@ export function depTypeToClass(depType: string) {
       return undefined;
   }
 }
+
+export function depTypeToLabel(type: EdgeType) {
+  switch (type) {
+    case EdgeType.peer:
+      return 'Peer Dependency';
+    case EdgeType.dev:
+      return 'Development Dependency';
+    case EdgeType.runtime:
+      return 'Dependency';
+    default:
+      return (type as string).toLowerCase();
+  }
+}
```

Comment:
consider converting to object instead of switch case

Reply:
thought of it - better to have function